Design Patterns/Form Objects.md

```diff
@@ -127,13 +127,16 @@ class RegistrationsController < ApplicationController
   end
 
   def create
-    @form = RegistrationForm.new(params[:registration])
+    @form = RegistrationForm.new(registration_params)
     if @form.save
       redirect_to root_path
     else
       render :new
     end
   end
+  private
+  def registration_params
+    params.require(:regisration_form).permit(:full_name, :company_name, :phone, :email)

```

Comment:
I'd use `:registration` instead of `:regisration_form` here (watch out for the typo).